js/src/dom/data.js

```diff
@@ -24,7 +24,7 @@ const mapData = (() => {
         id++
       }
 
-      storeData[element.bsKey.id] = data
+      storeData.set(element.bsKey.id, data)
```

Comment:
The `element` can not be used directly here? 🤔 
Ref: https://github.com/twbs/bootstrap/issues/32123#issuecomment-730180864

Reply:
yep the element should be used here to the data

Reply:
Uhm that's a bit contradictory to this response https://github.com/twbs/bootstrap/issues/32123#issuecomment-728855901 but well

Reply:
haha yes sorry @alpadev but things can change 🤷